src/webapp/index.js

```diff
@@ -232,16 +253,18 @@ async function getEther () {
   if (!account) return
 
   var uri = `${window.location.href}v0/request`
-  var data = account
+  var data = {
```

Comment:
may have to json serialize this first. verify that the fetch api supports providing an object

Reply:
@kumavis just a quick search makes it seem like I may have to use `JSON.stringify()`, does that sound right?